<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLineEdit" name="palid_lineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="add_Contact">
     <property name="text">
      <string>Add Pals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="id_Self">
     <property name="text">
      <string>id:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="add_group">
     <property name="text">
      <string>Add Group</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="quit_button">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ip_Self">
     <property name="text">
      <string>ip:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <widget class="QLabel" name="icon_Self">
     <property name="text">
      <string>icon</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>Friends</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
